spec/models/card_spec.rb

```diff
@@ -61,11 +61,12 @@
     expect(card.translated_text).to eq('house')
   end
 
-  it 'create card errors OK' do
-    card = Card.create(original_text: 'дом', translated_text: 'house',
-                       user_id: 1, block_id: 1)
-    expect(card.errors.any?).to be false
-  end
+  # it 'create card errors OK' do
```

Comment:
? if test not passd you can fix it, or made xeit instead it. it will reminder you about not passed test.

Reply:
fixed